### DISCUSSION TITLE
Internationalization

### DISCUSSION THREAD

**Discussion by AndreM222** [ghostty-org/ghostty#3731] [Feature Requests, Ideas] [answered]
Hi,

great job with the terminal, loving it.

I wanted to ask if there was plan to add internationalization.

currently my pc is in Japanese but the stuff are only rendering in English

**Comment by jcollie** [comment]
Ghostty does not have any translations or localization. We've been discussing the best way to do that on [Discord](https://discord.com/channels/1005603569187160125/1320769007317684245).

**Comment by mitchellh** [comment, marked as answer]
Dupe #2357
